graylog2-server/src/main/java/org/graylog/events/notifications/NotificationTestData.java

```diff
@@ -30,7 +30,9 @@
 import org.graylog2.plugin.rest.ValidationResult;
 import org.graylog2.plugin.streams.Stream;
 
-class NotificationTestData {
+public class NotificationTestData {
+    public static final String TEST_NOTIFICATION_ID = "1234";
```

Comment:
maybe we could make that id a little more explicit. like `testing-testing-123` :)

Reply:
As this is not a valid ObjectId anyway I didn't think it's necessary to make it more obvious. But sure, I can do this ;)